src/freenas/usr/local/libexec/nas/generate_smb4_conf.py

```diff
@@ -1004,6 +1004,10 @@ def generate_smb4_conf(client, smb4_conf, role):
     else:
         confset1(smb4_conf, "logging = file")
 
+    
```

Comment:
There are two line breaks here which will trip the PEP8 automated verification ;)